### ISSUE TITLE
train on large dataset

### ISSUE THREAD

**Issue by qiqiguaitm** [wy1iu/sphereface#29]
have you train your a-softmax on a much more dataset for example msceleb1m.
I trained a-softmax on a 10m dataset, abount 200k identities, the loss did not converge,
but softmax or center loss done well.

**Comment by wy1iu** [comment]
please see [issue 14](https://github.com/wy1iu/sphereface/issues/14)

**Comment by wy1iu** [comment]
I am closing this issue. You could discuss this topic there. Thanks.
